### PR TITLE
Tomek/wallet chooser tabs

### LINKED ISSUE #102: Create new wallet selector tabs style
It should have active state working.
image

### PR DESCRIPTION
**Pull request by banciur**
screencapture-localhost-9090-login-light-create-1518371065447

closes #102

### CODE REVIEW

File: app/components/walletSelector/WalletSelector.module.scss [hunk 1]
```diff
@@ -1,62 +1,35 @@
...
-.wallet-selector {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  align-self: center;
-  margin-top: 1.25rem;
+.wallet-chooser {
+  font-size: $smaller-font-size;
+  letter-spacing: -.2px;
+  margin-top: 1.563rem;
```
Comment (krzkaczor) [inline app/components/walletSelector/WalletSelector.module.scss]: same comments about margin. use `$spacing * multiplier` and roundup multplier.

File: app/components/walletSelector/WalletSelector.tsx [hunk 2]
```diff
@@ -48,26 +35,23 @@ const WalletTab = withRouter<IWalletTabLink>(WalletTabLink);
...
-            </WalletTab>
-          </div>
-        </div>
+      <Col
+        className={cn(
+          "d-flex flex-column flex-md-row justify-content-center mt-4 mb-5",
+          styles.walletChooser,
+        )}
+      >
+        <WalletTab href={walletRoutes.light} data-test-id="wallet-selector-light">
```
Comment (krzkaczor) [inline app/components/walletSelector/WalletSelector.tsx]: switch to `<Link /> `

File: app/components/walletSelector/WalletSelector.tsx [hunk 3]
```diff
@@ -48,26 +35,23 @@ const WalletTab = withRouter<IWalletTabLink>(WalletTabLink);
...
+          <span className={cn("mb-3 mb-md-0", styles.wallet)}>use Neufund wallet</span>
+        </WalletTab>
+        <span className={cn("mx-3", styles.separator)} />
```
Comment (krzkaczor) [inline app/components/walletSelector/WalletSelector.tsx]: get rid of separator and use css selectors instead (content)

File: app/styles/global.scss [hunk 4]
```diff
@@ -2,4 +2,6 @@
 
 body {
   background-color: $pale-grey-50;
+  font-family: "Open Sans", sans-serif;
```
Comment (krzkaczor) [inline app/styles/global.scss]: overwrite bootstrap variables instead

File: app/components/walletSelector/WalletSelector.tsx [hunk 5]
```diff
@@ -21,20 +21,23 @@ export const WalletSelector: React.SFC = () => (
           className={cn("mb-3 mb-md-0", styles.wallet)}
           to={walletRoutes.light}
           data-test-id="wallet-selector-light"
+          data-text="use Neufund wallet"
```
Comment (banciur) [inline app/components/walletSelector/WalletSelector.tsx]: Why won't we use `title` attribute here. It's a part of html standard and in this case fits quite good

Comment (krzkaczor) [reply app/components/walletSelector/WalletSelector.tsx]: https://www.searchenginejournal.com/how-to-use-link-title-attribute-correctly/ The title is not meant to be a duplication of the anchor text. I think it's just a workaround and should be ignored entirely by browser thus using data-text

**Pull request merged**